CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+ 
+Thank you for considering contributing to Symfony UX! 
+
+Symfony UX is an open source, community-driven project, and we are happy to receive contributions from the community!
+
+> [!TIP]
+> It's a good idea to read the [Symfony's Contribution Guide](https://symfony.com/doc/current/contributing/index.html) first, even if not all of it applies to Symfony UX and should be adapted to this project (e.g.: Symfony UX has only one base branch, `2.x`).
+
+## Reporting an issue
+
+If you either find a bug, have a feature request, or need help/have a question, please [open an issue](https://github.com/symfony/ux/issues/new/choose).
+
+Please follow the issue template and provide as much information as possible,
```

Comment:
```suggestion
Please provide as much information as possible,
```

As i'm not sure everyone knows what's an issue template ?

Reply:
You're right, there is even no issue templates :p